src/main/scala/is/hail/keytable/KeyTable.scala

```diff
@@ -888,4 +888,25 @@ case class KeyTable(hc: HailContext, rdd: RDD[Row],
 
     sb.result()
   }
+
+  def maximalIndependentSet(iExpr: String, jExpr: String): Array[Any] = {
+    val ec = EvalContext(fields.map(fd => (fd.name, fd.typ)): _*)
+
+    val (iType, iThunk) = Parser.parseExpr(iExpr, ec)
+    val (jType, jThunk) = Parser.parseExpr(jExpr, ec)
+
+    if (iType != jType)
+      fatal(s"the node expressions must have the same type: type of `i' is $iType, but type of `j' is $jType")
```

Comment:
"the node" => "Node" or "node"

Reply:
I've made the change but I don't understand what's wrong with using "the". I'm referring specifically to the node expressions passed to this function not all node expressions of which one could conceive.